internal/dataplane/parser/parser.go

```diff
@@ -121,14 +118,9 @@ func (p *Parser) Build() (*kongstate.KongState, error) {
 	result.Certificates = mergeCerts(p.logger, ingressCerts, gatewayCerts)
 
 	// populate CA certificates in Kong
-	var err error
-	caCertSecrets, err := p.storer.ListCACerts()
-	if err != nil {
-		return nil, err
-	}
```

Comment:
It didn't make sense to keep this behaviour - that was the only leftover in this function that could potentially return an error while we're already ignoring other errors that could be returned from `p.storer.List*` or `p.storer.Get*` functions. In the next steps, we should introduce a common way to report errors instead of just logging them.